services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -81,10 +81,27 @@ export const DashboardPage = () => {
         >
           <AccordionItem label="Instructions" sx={sx.accordionItem}>
             <Box sx={sx.accordionPanel}>
-              <p>
-                [Optional - Include instructions that would support the state in
-                the completion of the report]
-              </p>
+              <Heading size="sm" fontWeight="bold">
+                Admin Instructions
```

Comment:
I believe these instructions should be admin only, right now a state user sees them as well